cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Node.kt

```diff
@@ -238,6 +262,11 @@ open class Node : IVisitable<Node>, Persistable, LanguageProvider, ScopeProvider
         prev.nextDFG.add(this)
     }
 
+    fun addPrevCDG(prev: Node) {
+        prevCDGEdges.add(PropertyEdge(this, prev))
+        prev.nextCDGEdges.add(PropertyEdge(prev, this))
```

Comment:
The edge in both nodes should have the same start and end
```suggestion
        val edge = PropertyEdge(prev, this)
        prevCDGEdges.add(edge)
        prev.nextCDGEdges.add(edge)
```